### ISSUE TITLE
ToC format

### ISSUE THREAD

**Issue by alis0nc** [alis0nc/kuthesis#8]
There's something slightly wrong with the ToC indentation -- it should be on 1/2" tab stops. Like so:

```
-0.5"->|
I.      INTRODUCTION ......................
------1"------>|Problem Topic .................
                Background ...................
II.     CONCLUSIONS AND RECOMMENDATIONS ........
-0.5"->|
```

Also, the filler dots in the ToC items should be more closely spaced. `.........` versus `. . . . .`

**Comment by alis0nc** [comment]
Dots closeinified in cc7bb8b

**Comment by alis0nc** [comment]
ToC indents fixed in 7b04db9
